### DISCUSSION TITLE
A way to adjust the speed of the video while transcribing?

### DISCUSSION THREAD

**Discussion by richardcave27** [BigSoftVideo/DOTE#6] [Q&A] [answered]
Hi is there a way to slow down the video as I transcribe in Dote? I cannot see a way to do this. Thanks for any help

**Comment by skandilocks** [comment, marked as answer]
Hi Richard, no, there isn't at present in DOTE v1.0.2. A few people have asked about this, so we may add this sort of functionality (modifying playback speed etc) in a minor release in 2023.

Thanks for the question.

**Reply by richardcave27** [reply]
Thanks so much.
